api/pom.xml

```diff
@@ -71,7 +71,7 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpmime</artifactId>
-            <version>${http-components.version}</version>
+            <version>${bt-httpmime.version}</version>
```

Comment:
These properties are named `bt-httpclient-version` and `bt-httpmime-version`, without periods.